frontend/src/pages/home.tsx

```diff
@@ -0,0 +1,33 @@
+import {useSession} from 'next-auth/react';
+import {useEffect} from 'react';
+import {getMe, useAuthenticatedApi} from '@/api/api';
+import {User} from '@/api/models';
+import {useRouter} from 'next/router';
+
+export default function Home() {
+    const {data: session} = useSession();
+
+    const [userData, setUserData] = useAuthenticatedApi<User>();
+
+    const router = useRouter();
+
+    useEffect(() => {
+        if (session) {
+            // @ts-ignore
+            getMe(session, setUserData);
+        }
+    }, [session]);
+
+    useEffect(() => {
+        if (userData && userData.data) {
+            if (userData.data.student && !userData.data.student.is_super_student) {
+                router.push('/my-schedule');
+            } else if (userData.data.syndicus) {
+                router.push('/my-buildings');
+            } else {
+                router.push('/live_routes');
+            }
+        }
+    }, [userData]);
+    return (<div></div>);
```

Comment:
Is this supposed to just be empty?

Reply:
Yes, the only function of this page is to redirect after login. The callback from next-auth does not support dynamic routes, so we have to redirect after the redirect from the login.

So the user goes to /login -> redirected to /home which redirects to the correct starting page for the user for example /my-buildings or /live-routes

Reply:
I see, and what about the TODO's in certain files?

Reply:
This is because the page is not implemented yet. There is no my-buildings yet, and no my-schedule (but this will be added once I update @jchetti's PR and add his integration in this routing)

Reply:
Alright, so then these are okay to be merged in dev for now?

Reply:
Yes is functional on it's own, but https://github.com/SELab-2/Dr-Trottoir-3/pull/251 will have to be updated before merge